README.md

```diff
@@ -97,6 +97,10 @@ export class HomeComponent implements OnInit {
 }
 ```
 
+## Server-side rendering (SSR) with Angular Universal
+
+Library tested with SSR scenario you can check the demo.
```

Comment:
Something like would be better: The library is tested with an SSR scenario. Demo is located in the todo: directory.